deploy-sandpit.sh

```diff
@@ -52,7 +52,7 @@ pushd "${API_DIR}/ci/terraform/shared" > /dev/null
 PRIVATE_KEY="$(terraform output -json | jq -r 'first(.stub_rp_client_credentials.value[] | select (.client_name == $client_name)).private_key' --arg client_name "${CLIENT_NAME}")"
 
 cat <<EOF > "/tmp/sandpit-vars.yml"
-op_base_url: https://api.sandpit.auth.ida.digital.cabinet-office.gov.uk
+op_base_url: https://oidc.sandpit.account.gov.uk/
```

Comment:
Low impact comment: should this have a trailing slash.
Disregard if yes.

Reply:
unfortunately yes, it doesn't match the issuer without so you get an error